packages/spor-react/src/loader/ProgressLoader.tsx

```diff
@@ -56,79 +57,78 @@ type ProgressLoaderProps = BoxProps & {
  * <ProgressLoader value={50} aria-label="Fetching your trips..." />
  * ```
  */
-export const ProgressLoader = ({
-  value,
-  label,
-  labelRotationDelay = 5000,
-  "aria-label": ariaLabel,
-  width,
-  ...rest
-}: ProgressLoaderProps) => {
-  const { t } = useTranslation();
-  const currentLoadingText = useRotatingLabel({
-    label,
-    delay: labelRotationDelay,
-  });
-  const { labelProps, progressBarProps } = useProgressBar({
-    isIndeterminate: value === undefined,
+
+export const ProgressLoader = forwardRef<HTMLDivElement, ProgressLoaderProps>(
+  ({
     value,
-    "aria-label": ariaLabel ?? t(texts.fallbackLabel(value ?? "?")),
-  });
-  const pathRef = useRef<SVGPathElement>(null);
-  const progressPathLength = pathRef.current?.getTotalLength() ?? 0;
-  const progress = ((value - 100) / 100) * progressPathLength;
-  const id = useId();
-  return (
-    <Box {...progressBarProps} minWidth="100px" width={width} {...rest}>
-      <Box as="svg" viewBox="0 0 246 78" fill="none">
```

Comment:
litt usikker her på hva/om det ødelegger å fjerne viewBox. Tar gjerne imot tips til hvordan den kan legges inn på annen måte, for slik den var nå fikk den ikke være